x-pack/plugins/searchprofiler/common/constants.ts

```diff
@@ -0,0 +1,13 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { LicenseType } from '../../licensing/common/types';
```

Comment:
Right, so that value is being pulled in from the `common` folder in the licensing plugin. They have marked that type public which is way I am reaching in to that plugin to pull it out.

Perhaps worth opening a question about it to either have licensing explain the reasoning or update the platform doc to cater for values that are not public or server specific.

I will add an `index.ts` file to `common` inside search profiler marking these values similarly as internal :+1: